### ISSUE TITLE
News and Interests Not working and Microsoft Edge new tab content not loading

### ISSUE THREAD

**Issue by celphogeek** [0x6d69636b/windows_hardening#64]
Hi,

I import some settings from this hardening project and I have been reset all the configurations and import my backup too. But, news and interests it's not working and also microsoft edge stable are not loading the new tab content.

Obs: Edge Canary loading the new tab but the stable didn't load, not sure why :( .

I reset the group policy too but I check the regedit and it seems that some settings it's not removed, could someone try to help me ? I read all the settings that I apply and I'm testing but this is the 2 problems that is taking time to find out. Maybe spotlight is related with this?
![image](https://user-images.githubusercontent.com/29079236/128649385-f1b0cb8a-7ca9-4aee-bb07-87b2a2749da9.png)
![image](https://user-images.githubusercontent.com/29079236/128649398-e9df6c19-e7eb-4b72-9bcc-699011b66d13.png)

And here's the Edge Canary load the content

![image](https://user-images.githubusercontent.com/29079236/128649420-cbc5e2df-92f0-4780-bec7-f5c56de39d51.png)

**Comment by celphogeek** [comment]
The hardening list I apply was
finding_list_bsi_sisyphus_windows_10_hd_machine.csv
finding_list_bsi_sisyphus_windows_10_hd_user.csv
finding_list_0x6d69636b_user.csv
finding_list_0x6d69636b_machine.csv
finding_list_cis_microsoft_windows_10_enterprise_2004_machine.csv
finding_list_cis_microsoft_windows_10_enterprise_2004_user.csv
finding_list_microsoft_windows_tls.csv
finding_list_microsoft_windows_tls_future.csv
finding_list_msft_security_baseline_edge_92_machine.csv

**Comment by 0x6d69636b** [comment]
I do not recommend using different lists/standards such as my list, CIS and BSI SiSyPHuS on the same system. Sometimes the lists have different targets (e.g. PowerShell) and there may be undesirable side effects.

Now, I suspect it could be one of these user settings:

- finding_list_bsi_sisyphus_windows_10_hd_user: ID 110
- finding_list_0x6d69636b_user.csv: 4201

If it wasn't one of these settings, I would first find out if it was a machine or user setting. The easiest way to find out if it is a machine or user setting is to create a new profile, and if it works there, then it is a user setting. And then you have to narrow it down.

**Comment by celphogeek** [comment]
will create a new user and them check it out

**Comment by celphogeek** [comment]
create new user and the edge stable is working 
![image](https://user-images.githubusercontent.com/29079236/128659132-c6f46209-4cf2-4190-b620-3fb5e7735db1.png)

I don't see the news and interest on taskbar, I will restart the computer to take a look on this.

**Comment by celphogeek** [comment]
Restart the computer and news and interests not appear in the new user profile created, I'm checking all the settings applied to find a possible cause for the issue.

**Comment by 0x6d69636b** [comment]
The News and Interests has it's own config (which is not managed by any the lists yet), maybe it helps to check for this in the registry: 

- HKEY_CURRENT_USER\Software\Microsoft\Windows\CurrentVersion\Feeds -> ShellFeedsTaskbarViewMode (2 = disabled)
- HKEY_LOCAL_MACHINE\SOFTWARE\Policies\Microsoft\Windows\Windows Feeds -> EnableFeeds (0 = disabled)

**Comment by 0x6d69636b** [comment]
Just to let you know, I added now a setting to the list _finding_list_0x6d69636b_machine_, but I recommend to disable the setting. So if you use the newest version of the list, you have to correct the setting to 1 (ID 1761).

**Comment by celphogeek** [comment]
Check here and EnableFeeds I already change to 1 and ShellFeedsTaskbarViewMode is set to 0
![image](https://user-images.githubusercontent.com/29079236/128755205-02a28ea7-7550-43c7-974e-6d9b9abccd8d.png)
![image](https://user-images.githubusercontent.com/29079236/128755246-1c0ebcf0-60d4-4d79-97fe-ca7a895dbf36.png)

**Comment by celphogeek** [comment]
Not sure if this error from Microsoft Store is related, but maybe it is
![image](https://user-images.githubusercontent.com/29079236/128757802-9bf352e4-d669-4877-a574-227477b46533.png)

I'm looking all the settings that have been applied and try to figure out which one cause this issue

**Comment by celphogeek** [comment]
I have applied the following settings manually in gpedit.msc
- finding_list_bsi_sisyphus_windows_10_hd_user: ID 110
- finding_list_0x6d69636b_user.csv: 4201

Also I applied, Internet communication (HKCU and HKLM) and just let not configured the store option and windows update,RSS Feeds and other settings that I have in mind and it seems News and Interests working.

Obs: I make a clean installation of windows to find the cause of the issue, until now nothing. Need to see if the changes made in gpedit.msc will have a different effect in Regedit (using the same settings).
